client/src/components/NewVersionAvailable.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Dialog from './Dialog';
+import css from './NewVersionAvailable.css';
+
+const NewVersionAvailable = ({ newVersionAvailable }) => (
+  <Dialog
+    onClose={() => { window.location.href = '.'; }}
+    title="En ny versjon av applikasjonen er tilgjengelig"
```

Comment:
This might be a bit long. There's no space between the title and corner button. I'd suggest adding some margin-left to the corner in the Card class, should alleviate the problem and future occurrences.